app/src/main/res/values-ca/strings.xml

```diff
@@ -89,6 +89,21 @@
     <string name="calendar_select_date">Selecciona una data</string>
     <string name="add_personal_event_button">Personal</string>
     <string name="add_periodic_event_button">Periòdic</string>
+    <string name="dialog_new_event">Nou event personal</string>
+    <string name="dialog_new_event_message">Introdueix la descripció de l\'event, l\'hora i selecciona la mascota que participarà.</string>
```

Comment:
"Event" is incorrect in catalan. The translation from spanish would be "Esdeveniment".